litellm-proxy-groq/README.md

```diff
@@ -0,0 +1,114 @@
+# Use LiteLLM Proxy to Call Groq AI API
+
+Use [LiteLLM Proxy](https://docs.litellm.ai/docs/simple_proxy) for:
+- Calling 100+ LLMs Groq AI, OpenAI, Azure, Vertex, Bedrock/etc. in the OpenAI ChatCompletions & Completions format
+- Track usage + set budgets with Virtual Keys
+
+Using [Groq AI API with LiteLLM](https://docs.litellm.ai/docs/providers/groq)
+
+## Sample Usage
+
+### Step 1. Create a Config for LiteLLM proxy
+
+LiteLLM Requires a config with all your models define - we can call this file `litellm_config.yaml`
+
+[Detailed docs on how to setup litellm config - here](https://docs.litellm.ai/docs/proxy/configs)
+
+```yaml
+model_list:
+  - model_name: groq-llama3 ### MODEL Alias ###
+    litellm_params: # all params accepted by litellm.completion() - https://docs.litellm.ai/docs/completion/input
+      model: groq/llama3-8b-8192 ### MODEL NAME sent to `litellm.completion()` ###
+      api_key: "os.environ/GROQ_API_KEY" # does os.getenv("GROQ_API_KEY")
+
+```
+
+### Step 2. Start litellm proxy
+
+```shell
+docker run \
+    -v $(pwd)/litellm_config.yaml:/app/config.yaml \
+    -e GROQ_API_KEY=<your-groq-api-key>
+    -p 4000:4000 \
+    ghcr.io/berriai/litellm:main-latest \
+    --config /app/config.yaml --detailed_debug
+```
+
+### Step 3. Test it! 
+
+[Use with Langchain, LlamaIndex, Instructor, etc.](https://docs.litellm.ai/docs/proxy/user_keys)
+
+```bash
+import openai
```

Comment:
In the [Docs you link to](https://github.com/groq/groq-api-cookbook/blob/b89d34f4a889edaeb99f314f6b399ce7962903a8/litellm-proxy-groq/README.md) the code is different (`from litellm import completion` instead of the OpenAI package). Are both usages valid?

Reply:
<img width="867" alt="Screenshot 2024-07-24 at 2 36 31 PM" src="https://github.com/user-attachments/assets/355a0a89-0eb3-4e3f-85f3-ed0d2e847b8d">

just updated the doc on litellm proxy to show how to do this too

Reply:
https://docs.litellm.ai/docs/providers/groq#usage-with-litellm-proxy